x11-libs/wxGTK/wxGTK-3.2.1.ebuild

```diff
@@ -0,0 +1,182 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
This only inherits multilib-minimal, so we should be able to bump to EAPI 8 with this new slot?

Reply:
Yes, sounds right.